### DISCUSSION TITLE
How to run example

### DISCUSSION THREAD

**Discussion by winionian** [tr3v3r/react-native-esc-pos-printer#189] [Q&A]
I has do command 'git clone' and 'npm install' to test example code but not success to build on Android device.
There are many step that use to do before build the example application ?

**Comment by tr3v3r** [comment]
Need more details regarding  - "not success" please.

generally

1) git clone
2) yarn
3) cd example
4) yarn run android

**Reply by winionian** [reply]
===== Build successfull but show below on node metro



Last login: Fri Oct 25 19:26:03 on ttys011
/Users/pawinsanvongsiri/Desktop/react_native/react-native-esc-pos-printer/example/node_modules/.bin/launchPackager.command ; exit;
pawinsanvongsiri@MacBook-Pro-khxng-pawin ~ % /Users/pawinsanvongsiri/Desktop/react_native/react-native-esc-pos-printer/example/node_modules/.bin/launchPackager.command ; exit;
error Cannot read properties of undefined (reading '100').
TypeError: Cannot read properties of undefined (reading '100')
    at forEachCode (/Users/pawinsanvongsiri/Desktop/react_native/react-native-esc-pos-printer/example/node_modules/http-errors/index.js:261:45)
    at Array.forEach (<anonymous>)
    at populateConstructorExports (/Users/pawinsanvongsiri/Desktop/react_native/react-native-esc-pos-printer/example/node_modules/http-errors/index.js:259:9)
    at Object.<anonymous> (/Users/pawinsanvongsiri/Desktop/react_native/react-native-esc-pos-printer/example/node_modules/http-errors/index.js:31:1)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:121:18)
info Run CLI with --verbose flag for more details.
Process terminated. Press <enter> to close the window




======== the "package.json" file on example directory


<img width="1023" alt="image" src="https://github.com/user-attachments/assets/7597201e-13f8-4e64-9808-514daf31b69b">

<img width="431" alt="image" src="https://github.com/user-attachments/assets/5320d7a2-950e-4680-91e4-5fb07c6cefdc">

========= react-native info
pawinsanvongsiri@MacBook-Pro-khxng-pawin example % react-native info
info Fetching system and libraries information...
System:
  OS: macOS 14.4.1
  CPU: (8) x64 Apple M1
  Memory: 14.52 MB / 8.00 GB
  Shell:
    version: "5.9"
    path: /bin/zsh
Binaries:
  Node:
    version: 18.17.1
    path: ~/.nvm/versions/node/v18.17.1/bin/node
  Yarn:
    version: 3.6.1
    path: /opt/homebrew/bin/yarn
  npm:
    version: 10.8.2
    path: ~/.nvm/versions/node/v18.17.1/bin/npm
  Watchman: Not Found
Managers:
  CocoaPods:
    version: 1.15.2
    path: /opt/homebrew/bin/pod
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 23.4
      - iOS 17.4
      - macOS 14.4
      - tvOS 17.4
      - visionOS 1.1
      - watchOS 10.4
  Android SDK: Not Found
IDEs:
  Android Studio: 4.2 AI-202.7660.26.42.7486908
  Xcode:
    version: 15.3/15E204a
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 11.0.16.1
    path: /opt/homebrew/opt/openjdk@11/bin/javac
  Ruby:
    version: 3.3.0
    path: /opt/homebrew/opt/ruby/bin/ruby
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.72.6
    wanted: 0.72.6
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: false
iOS:
  hermesEnabled: true
  newArchEnabled: false

info React Native v0.76.0 is now available (your project is running on v0.72.6).
info Changelog: https://github.com/facebook/react-native/releases/tag/v0.76.0
info Diff: https://react-native-community.github.io/upgrade-helper/?from=0.72.6
info For more info, check out "https://reactnative.dev/docs/upgrading?os=macos&platform=android".
